ks-svarut/src/main/java/no/difi/meldingsutveksling/ks/svarinn/SvarInnMessage.java

```diff
@@ -86,6 +86,16 @@ private JournpostType createJournpostType() {
         journpostType.setJpJaar(metadata.getJournalaar());
         journpostType.setJpSeknr(metadata.getJournalsekvensnummer());
         journpostType.setJpJpostnr(metadata.getJournalpostnummer());
+        journpostType.setJpOffinnhold(metadata.getTittel());
+        journpostType.setJpJdato(metadata.getJournaldato());
+        journpostType.getAvsmot().add(CreateSaksbehandlerAvsernder(metadata));
         return journpostType;
     }
+
+    private AvsmotType CreateSaksbehandlerAvsernder(Forsendelse.MetadataFraAvleverendeSystem metadata){
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS00100)